lib/console.js

```diff
@@ -24,8 +24,14 @@ function Console(stdout, stderr) {
   Object.defineProperty(this, '_stdout', prop);
   prop.value = stderr;
   Object.defineProperty(this, '_stderr', prop);
+  prop.value = ignoreErrors === undefined ? true : !!ignoreErrors;
```

Comment:
That would have a different default behaviour, though…? `!!undefined` is `false`.